prime-dotnet-webapi/Controllers/EnrolleesController.cs

```diff
@@ -70,6 +70,21 @@ public async Task<ActionResult> GetEnrollees([FromQuery] EnrolleeSearchOptions s
             }
         }
 
+        // GET: api/Enrollees/PotentialPaperEnrolleeReturneeStatus/5
+        /// <summary>
+        /// Gets all paper enrollees and checks whether or not there is a match in dateOfBirth with the current logged on enrollee.
+        /// </summary>
+        [HttpGet("PotentialPaperEnrolleeReturneeStatus/{dateOfBirth}")]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<Enrollee>>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<EnrolleeListViewModel>>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> GetPotentialPaperEnrolleeReturneeStatus(string dateOfBirth)
```

Comment:
You can make the parameter a `DateTime` rather than a string and ASP will de-serialize it directly for you without having to parse it yourself from a string.

Reply:
Changed to DateTime